core/AtlasState.hpp

```diff
@@ -52,6 +54,9 @@ namespace atlas
         static constexpr char name[] = "AtlasState";
         using base_type = AtlasState;
 
+        //STFLogger Object
+        atlas::STFLogger stf_logger_;
```

Comment:
Private variable in public space.  Should be accessible via public interface methods only.